netlify.toml

```diff
@@ -5,7 +5,13 @@
 [build]
   publish = "public"
   base = "website"
-  command = "cd themes/docsy && git submodule update -f --init && cd ../.. && npm install postcss-cli autoprefixer@^9.0.0 && hugo version && hugo"
+  command = """
+git submodule update -f --init themes/docsy && \
+npm install postcss-cli autoprefixer@^9.0.0 && \
+./scripts/set_version.sh && \
```

Comment:
@asmacdo this script updates `version_menu = "Releases"` to `version_menu = "vX.Y"` at build time if the branch matches `vX.Y.x`. This should render the updated menu as it currently is now, but without us having to commit this change, right?